nexus/db-model/src/schema.rs

```diff
@@ -13,7 +13,7 @@ use omicron_common::api::external::SemverVersion;
 ///
 /// This should be updated whenever the schema is changed. For more details,
 /// refer to: schema/crdb/README.adoc
-pub const SCHEMA_VERSION: SemverVersion = SemverVersion::new(30, 0, 0);
+pub const SCHEMA_VERSION: SemverVersion = SemverVersion::new(31, 0, 0);
```

Comment:
You need to be 32 so https://github.com/oxidecomputer/omicron/pull/4852 can be 31